src/main/java/org/fundacionjala/pivotal/common/CommonMethods.java

```diff
@@ -0,0 +1,40 @@
+package org.fundacionjala.pivotal.common;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * Class CommonMethods.
+ */
+public final class CommonMethods {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)